$OpenBSD: patch-igor_1,v 1.3 2017/11/29 13:26:36 schwarze Exp $

Chunks 1+3: Do not report contractions by default (local change).
Chunks 2+4: Do not use /usr/local/etc/ (local change).

Index: igor.1
--- igor.1.orig
+++ igor.1
@@ -104,7 +104,7 @@ When ready for the actual commit, run
 .Nm
 without this option to check the date.
 Equivalent to
-.Fl abefmrsuw .
+.Fl abefmrsw .
 .It Fl z
 Run all applicable non-whitespace
 .Pq content
@@ -154,7 +154,7 @@ documents.
 Additional files of misspelled words may be added with the
 .Ev IGORSPELLFILES
 environment variable, or placed in the
-.Pa /usr/local/etc/igor/spelling
+.Pa /etc/igor/spelling
 directory.
 The first word on each line, including apostrophes, will be included in the
 list of known misspellings to be checked.
@@ -163,6 +163,12 @@ The Debian project has a list of words at
 .Li http://anonscm.debian.org/gitweb/?p=lintian/lintian.git;a=blob_plain;f=vendors/debian/ftp-master-auto-reject/data/spelling/corrections;hb=HEAD
 .It Fl u
 Identify contractions, which make translation more difficult.
+This test is off by default on
+.Ox
+because
+.Ox
+documentation maintainers regard contractions as a normal feature
+of the English language and see no need to avoid them.
 .It Fl w
 Find general whitespace errors, including trailing whitespace at
 the end of lines, blank lines containing whitespace, and tabs
@@ -271,8 +277,8 @@ A list of whitespace-delimited filenames containing ad
 words.
 .El
 .Sh FILES
-.Bl -tag -width /usr/local/etc/igor/spelling/
-.It Pa /usr/local/etc/igor/spelling/
+.Bl -tag -width /etc/igor/spelling/
+.It Pa /etc/igor/spelling/
 Directory for additional misspelled word files.
 .El
 .Sh EXAMPLES
